diff_result.py

```diff
@@ -0,0 +1,30 @@
+from collections import defaultdict
+class DiffResult:
+    def __init__(self):
+        self.files = []
+        self.regions = 0
+        self.lineAdded = 0
+        self.lineDeleted = 0
+        self.functionCalls = defaultdict(int)
+
+    def toText(self):
+        with open('diffResult.txt', 'w') as output:
```

Comment:
This could have been an option to the commandline, but it's fine this way too

Reply:
A valid suggestion, thank you